components/ws-manager/go.mod

```diff
@@ -73,6 +73,8 @@ require (
 	github.com/inconshreveable/mousetrap v1.0.0 // indirect
 	github.com/json-iterator/go v1.1.12 // indirect
 	github.com/klauspost/cpuid v1.3.1 // indirect
+	github.com/klauspost/pgzip v1.2.5 // indirect
+	github.com/kubernetes-csi/external-snapshotter/client/v6 v6.0.0-rc4 // indirect
```

Comment:
> Also, the min Kubernetes verion for snapshotter is 1.20. What's our promised min Kubernetes version for self-hosted solution?

Okay, I found the answer https://www.gitpod.io/docs/self-hosted/latest/cluster-set-up#supported-kubernetes-distributions Our min Kubernetes version for self-hosted solution is 1.21.

Reply:
It doesn't seem like v5 is available:
```
go get -u github.com/kubernetes-csi/external-snapshotter/client/v5
go: module github.com/kubernetes-csi/external-snapshotter@upgrade found (v1.2.2), but does not contain package github.com/kubernetes-csi/external-snapshotter/client/v5
```
unless I am doing something wrong. :thinking: 
Though `v4` works, so I will switch to that one.

Reply:
Let me check 👨‍💻 
Since we install snapshot-controller v5.0.1, it'd be better we use the same version 😄

Reply:
Okay, the problem is the client/v5.0.0 released with v4 version https://github.com/kubernetes-csi/external-snapshotter/blob/client/v5.0.0/client/go.mod#L1. Ref to https://github.com/kubernetes-csi/external-snapshotter/pull/670#discussion_r833332402

I think we should downgrade the snapshot controller and the CRD version to v4.2.x (either v4.2.0 or v4.2.1) in workspace repo.

Reply:
I think version of the client doesn't matter here. Why?
We are using GA version of the CRD, and I use the client here only as a way to interact with CRD easier (so that I don't need to use UnstructuredData to create\update k8s object). 
We don't use client for anything else.

Reply:
I see.

I check the kubernetes-csi/external-snapshotter releases, looks like v4 is the main focus released (having lots of major/minor release). I feel like v5 is an intermediate release. Do you think we should keep the snapshot-controller as it is _or_ keep the snapshot-controller major/minor version matches to the client version?

Reply:
Okay, let's keep is as is.